fluent/protocol/packed_forward_message.go

```diff
@@ -199,8 +214,9 @@ func NewCompressedPackedForwardMessageFromBytes(
 	tag string, entries []byte,
 ) (*PackedForwardMessage, error) {
 	mc := compressorPool.Get().(*GzipCompressor)
+	mc.Reset()
```

Comment:
I moved it so that the patterns are consistent between the various pooled types. It shouldn't make much difference when it is cleared. If traffic is high, then the instance will be reused often. If it low, then there won't be many instances sitting around.

Reply:
Agreed